HLeithner/joomla-cms
# Add prepared statements for plg_fields-sql

This PR tries to move the SQL custom field into a prepared statement, at least for the values part.

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions

* Create a SQL custom field
* Use several SQL queries in the custom field
* check the frontend if the expected result could be found.



### Expected result
Nothing changed.
